pkg/ui/src/routes/login.tsx

```diff
@@ -1,12 +1,29 @@
 import React from "react";
 import { Route } from "react-router";
+import { Store } from "redux";
 
+import { doLogout, selectLoginState } from "src/redux/login";
+import { AdminUIState } from "src/redux/state";
 import LoginPage from "src/views/login/loginPage";
 
 export const LOGIN_PAGE = "/login";
+export const LOGOUT_PAGE = "/logout";
+
+export default function(store: Store<AdminUIState>): JSX.Element {
```

Comment:
nit: I don't like exporting anonymous functions; prefer to give them names (I think you can still export them in the same statement; maybe need an export statement below). Always giving functions names makes them easier to grep for and find via stack traces (as if we ever have reasonable stack traces… 😢 but that's a separate issue)

Reply:
Good point.  Done.